linkml_model/graphql/meta.graphql

```diff
@@ -625,6 +625,13 @@ interface SlotExpression
     allOf: [AnonymousSlotExpression]
   }
 
+type StructuredAlias
+  {
+    name: String!
+    source: Uriorcurie
```

Comment:
Will there be cases in which the source is identified simply with a string instead of a uri or curi?